Zotero/Controllers/Database/Requests/EditAnnotationFontSizeDbRequest.swift

```diff
@@ -13,7 +13,7 @@ import RealmSwift
 struct EditAnnotationFontSizeDbRequest: DbRequest {
     let key: String
     let libraryId: LibraryIdentifier
-    let size: UInt
+    let size: CGFloat
```

Comment:
I can't add comment below to "original content", but is `field.value = "\(self.size)"` ok when we changed type to `CGFloat`? Wouldn't it be better to round the value here?

Reply:
`"\(self.size)"` is formatted accordingly. The value passed to the request, is already rounded.